contrib/pgexporter.conf

```diff
@@ -0,0 +1,15 @@
+[pgexporter]
```

Comment:
Lets delete this file.

We need to have documentation in `doc/CONFIGURATION.md` about `metrics_path`